### DISCUSSION TITLE
[Windows] Get `HBITMAP` from Avalonia Bitmap

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#15954] [Q&A] [answered]
Can I get 'HBITMAP` from Avalonia Bitmap?

**Comment by maxkatz6** [comment, marked as answer]
Avalonia Bitmap is not backed by HBITMAP, so you can't get one from another.
You would need to get a byte array of pixels, or encoded stream, and create HBITMAP from that.
